homeassistant/components/homewizard/config_flow.py

```diff
@@ -126,18 +120,14 @@ async def async_step_zeroconf(
         self, discovery_info: ZeroconfServiceInfo
     ) -> ConfigFlowResult:
         """Handle zeroconf discovery."""
+
         if (
-            CONF_API_ENABLED not in discovery_info.properties
-            or CONF_PATH not in discovery_info.properties
```

Comment:
These properties are not available in `_homewizard._tcp`, and are not used down the line.